pkg/maps/policymap/policymap.go

```diff
@@ -72,55 +72,70 @@ type PolicyEntryDump struct {
 }
 
 func (key *policyKey) String() string {
+	trafficDirectionString := ""
+	// Make sure that this is in sync with pkg/policy:Egress/Ingress constants.
```

Comment:
Why not just place the `Egress`/`Ingress` traffic direction constants into a package that's closer to the BPF map layer and share them?